source/elements/oneTBB/source/flow_graph/indexer_node_cls.rst

```diff
@@ -60,7 +60,7 @@ a reference to a specific input port.
 Member types
 ------------
 
-* ``input_ports_type`` is an alias to a tuple of input ports.
+* ``input_ports_type`` is an alias to a ``std::tuple`` of input ports.
```

Comment:
Shouldn't we change it on line 46?
```
It contains a tuple of input ports, each of which is a
```

Reply:
I think that it is not necessary to add `std` everywhere within the spec of one node. It may be sufficient to indicate this one or more times within the return or input value types, and in others place to use the word tuple, as a concept term. What do you think about it?

Reply:
Ok, agree